examples/calibrationDataRead/calibrationDataRead.ino

```diff
@@ -0,0 +1,56 @@
+/*
+ * This example shows how to read the TDS and EC calibration values
+ * from Amperka TDS-sensor
+ * During the calibration, use standart buffer solution of know TDS value
+ * For more details read the calibration section in API.md
+ */
+
+// Include the library to communicate with Amperka TDS-sensor
+#include <AmperkaTDS.h>
+
+// Use any analog pin of the controller
+constexpr auto pinSensor = A0;
+
+// Create an object for working with Amperka TDS-sensor
+AmperkaTDS sensorTDS(pinSensor);
+
+void setup() {
+    // Open serial port with 9600 baud rate
+    Serial.begin(9600);
+    // Wait for serial port to connect, needed for native USB.
+    while (!Serial)
+        ;
+    // Start the sensor for communication
+    sensorTDS.begin();
+    // Calibrate the sensor
+    calibrateSensor();
+}
+
+void loop() {
+    // Read the sensor data values: EC and TDS
+    sensorTDS.read();
+    // Receive the stored electrical conductivity (EC)
+    int valueEC = sensorTDS.getEC();
+    // Receive the stored total dissolved solids (TDS)
+    int valueTDS = sensorTDS.getTDS();
+    // Print results
+    Serial.print("EC Value = ");
+    Serial.print(valueEC);
+    Serial.print(" uS/cm");
+    Serial.print("\t");
+    Serial.print("TDS Value = ");
+    Serial.print(valueTDS);
+    Serial.println(" ppm");
+    // Wait 1000 ms
+    delay(1000);
+}
+
+// Function for calibration sensor
+// For more details read the calibration section in API.md file.
+void calibrateSensor() {
```

Comment:
I think it's better to pass the calibration value to the function like this:
```suggestion
void calibrateSensor(int calibrationValue) {
```

Reply:
@AlexErAmp I thought about this option.
But then we use calibration value is 342 ppm in the main function (void setup).

``` cpp
void setup() {
    ...
    // Calibrate the sensor
    // For example buffer calibration solution is 342 ppm
    calibrateSensor(342);
}
```

Reply:
Doesn't it confuse the user? For example: _What is buffer calibration solution and value 342_? 
And in next library for PH-sensor, I use two calibration values. Where all process calibrate for user will be similar: in the calibration finction.

Maybe it's better when the all calibration process leave in a calibration function?

Reply:
By hiding the calibration value from the user inside a function, we confuse them even more. He may not look at the implementation of the function and call this function for calibration, not suspecting that he needs a 342 ppm solution :) Moreover, the calibration solution may have a different value.